test/buffer/worker/001_send_records_test.go

```diff
@@ -22,6 +25,26 @@ func (ts *testSuite) test001SendPayload() {
 		ts.SendPayload(i)
 	}
 
+	// Check API metrics
+	var recordsMetric float64
+	var bodyBytesMetric float64
+	ts.IterateMetrics(func(ch <-chan *dto.MetricFamily) {
+		for f := range ch {
+			switch *f.Name {
+			case "keboola_go_buffer_ingress_records_total":
+				for _, m := range f.Metric {
+					recordsMetric += m.Counter.GetValue()
+				}
+			case "keboola_go_buffer_ingress_bytes_total":
+				for _, m := range f.Metric {
+					bodyBytesMetric += m.Counter.GetValue()
+				}
+			}
+		}
+	})
+	assert.Equal(ts.t, float64(4), recordsMetric)
+	assert.Equal(ts.t, float64(84), bodyBytesMetric)
```

Comment:
Doplnene do worker E2E testu.

Tu som sa trocha potrapil. Kedze je to E2E test a bezi tam realne 5x API node, a request sa posle vzdy na nahodne vybranu API node.

Teda potrebujem nacitat z kazdej node `/metrics` endpoint, dekodovat ho a spocitat co potrebujem.
To sa lisi od telemetrie v unit testoch, kde mame na to priamo nastroje od OpenTelemetry.

Implementovat vsetky druhy metrik by bolo nerealne, preto si tu streamujem dekodovane metriky a spocitam si iba to co ma zaujima.